src/common/XChainIntegrator.sol

```diff
@@ -80,7 +93,6 @@ abstract contract XChainIntegrator is Auth {
 		}
 	}
 
-	// This one has to integrate with layerZero message sender
 	function startBridgeRoute(
```

Comment:
is this only used for testing? Seems like the chain vault would just use `sendTokens` directly right?